presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -133,6 +133,9 @@ class TaskManager {
   inline size_t getNumTasks() const {
     return taskMap_.rlock()->size();
   }
+  
+  /// Returns the processed input data size in bytes for tasks.
+  int64_t getBytesProcessed();
```

Comment:
@jkhaliqi : Make this a const function as well. (Since this other methods are marked that way)